rotkehlchen/tests/api/test_history_base_entry.py

```diff
@@ -118,9 +126,10 @@ def _add_entries(server) -> list[HistoryBaseEntry]:
 
 
 @pytest.mark.parametrize('number_of_eth_accounts', [0])
-def test_add_edit_delete_entries(rotkehlchen_api_server):
+@pytest.mark.parametrize('chain_id', [ChainID.ETHEREUM, ChainID.OPTIMISM])
```

Comment:
Do you really need to run the same identical test for 2 different chain IDs? Does it many any remote calls?

Reply:
Why not to have this test for optimism too?

It makes zero remote calls.

Reply:
If it makes zero remote calls it's fine. Did you run it with network checks on to confirm?

Reply:
I couldn't run it with `pytest-socket` since `pytest-socket` was also blocking places like this https://github.com/rotki/rotki/blob/f47ff8d26376ac49f302314444b31e3c1a16720f/rotkehlchen/tests/api/test_history_base_entry.py#L117, but I ran the test with WiFi turned off and it succeeded